src/app/modules/main/components/form-join-room/form-join-room.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+
+@Component({
+  selector: 'app-form-join-room',
+  templateUrl: './form-join-room.component.html',
+  styleUrls: ['./form-join-room.component.scss']
+})
+export class FormJoinRoomComponent implements OnInit {
+
+  public formJoinRoom: FormGroup;
+
+
+  constructor(private formBuilder: FormBuilder) {
+  }
+
+  ngOnInit(): void {
+    this.createForm();
+  }
+
+  public createForm(): void {
```

Comment:
I guess, this function can actually be no a function, just use that login in OnInit hook